src/shared/components/ChannelCover/ChannelCover.style.ts

```diff
@@ -44,88 +42,11 @@ export const CoverImage = styled.div<CoverImageProps>`
   background-position: center;
   background-attachment: local;
   background-size: cover;
```

Comment:
Cannot comment on the line below for some reason but cannot we make this a regular `img` now? Should be simpler than background for `div`